components/Banner.tsx

```diff
@@ -0,0 +1,37 @@
+import { SpeakerphoneIcon } from "@heroicons/react/outline";
+
+export default function Banner() {
+  return (
+    <>
+      <div className="bg-indigo-600">
+        <div className="max-w-7xl mx-auto py-3 px-3 sm:px-6 lg:px-8">
+          <div className="flex items-center justify-between flex-wrap">
+            <div className="w-0 flex-1 flex items-center">
+              <span className="flex p-2 rounded-lg bg-indigo-800">
+                <SpeakerphoneIcon
+                  className="h-6 w-6 text-white"
+                  aria-hidden="true"
+                ></SpeakerphoneIcon>
+              </span>
+              <p className="ml-3 font-medium text-white truncate">
+                <span className="md:hidden"> We need your help! </span>
+                <span className="hidden md:inline">
+                  The future of Evie is currently being made and we need your
```

Comment:
Really now

Reply:
forgot to delete that, oops

Reply:
https://github.com/ImageingApp/Website/commit/6243a5fbe1c3c37847abecea20f3078a37e7df27